assets/TLoadTest.txt

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
aaahh you meant this in txt?? I thought you were talking about the bytecode itself.. I find this a bit strange I have to admit 😬

Reply:
I would prefer to have this in a comment

Reply:
aaahh you rebased xD I thought you were talking about the bytecode 😅 sooryyy (because in my mind I had already thought about moving the bytecode of other stuff into files - I read what I want to read sometimes)